ci/vpp_test_setup/vpp_pod.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 
-CI_DIR="/runner/_work/userspace-cni-network-plugin/userspace-cni-network-plugin/ci/"
-kubectl apply -f $CI_DIR/vpp_test_setup/network_attachment_definition.yaml
-kubectl create -n vpp configmap vpp-app-startup-config --from-file=$CI_DIR/vpp_test_setup/startup.conf
+# set CI_DIR if not defined in parent script
```

Comment:
should a default just be set in parent dir instead?

Reply:
more modular for end user, allows user to run this script without the others